spec/compiler/CodeSectionSpec.js

```diff
@@ -57,6 +57,16 @@ describe("CodeSection", function() {
       section.appendDoc("world -->");
       expect(section.doc).toEqual("hello\nworld\n");
     });
+    it("does not trim code bloks", function() {
```

Comment:
Can you please add a test that ensure that the lines are trimmed outside of code blocks?